mne_connectivity/effective.py

```diff
@@ -243,3 +247,191 @@ def phase_slope_index(
         )
 
     return conn
+
+
+@verbose
+@fill_doc
+def phase_slope_index_time(data,
+                           names=None,
+                           indices=None,
+                           sfreq=2 * np.pi,
+                           mode="multitaper",
+                           fmin=None,
+                           fmax=np.inf,
+                           mt_bandwidth=None,
+                           freqs=None,
+                           n_cycles=7,
+                           padding=0,
+                           n_jobs=1,
+                           ):
+    """Compute the Phase Slope Index (PSI) connectivity measure across time rather than epochs.
```

Comment:
Just make sure you add a verbose parameter. That should get the tests running.

Reply:
Also you shouldn't need both verbose and fill_doc decorators. Verbose will trigger the same things as fill_doc

Reply:
@drammock Turns out `fill_doc` was needed since this is MNE-Connectivity's `fill_doc`, with access to MNE-Connectivity's `docdict`, whereas calling `verbose` only used the main MNE package's `fill_doc`.

Reply:
Ahh right, I don't think of that